src/sass/partials/_game-section.scss

```diff
@@ -0,0 +1,276 @@
+@use "./variables" as *;
+
+.your-card-title {
+  margin: 5% 0;
+  width: 100%;
+  display: block;
+}
+
+.game-container-icon {
+  position: absolute;
+  bottom: 5%;
+  left: 48%;
+}
+
+// GAME CONTAINER --------------------------------
+.game-container {
+  min-height: 100vh;
+  height: 100%;
+  padding: 10% 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  z-index: 20;
```

Comment:
You might want to look at having a consistent order to your properties.
I like this article that I found somewhere in the Juno repo:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/